src/main/java/com/example/onlinebookstore/security/jwt/JwtAuthenticationFilter.java

```diff
@@ -0,0 +1,54 @@
+package com.example.onlinebookstore.security.jwt;
+
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.stereotype.Component;
+import org.springframework.util.StringUtils;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+@Component
+@RequiredArgsConstructor
+public class JwtAuthenticationFilter extends OncePerRequestFilter {
+    private static final String AUTHORIZATION_HEADER = "Authorization";
```

Comment:
there's an existing constant - `HttpHeaders.AUTHORIZATION`

Reply:
i tried, but IDE I tried, but the IDE did not find such an import.
Could you tell me which import is needed, because I googled:
`import org.springframework.http.HttpHeaders;`
 but it does not exist

Reply:
Maybe  you lose 'static'
import static org.springframework.http.HttpHeaders.AUTHORIZATION;